doc/source/methods/ALE.ipynb

```diff
@@ -60,6 +60,16 @@
     "```python\n",
```

Comment:
<p>Move the paragraph below before the newly introduced paragraph for greater coherence?</p><p> </p><p>"This is useful if the number of total features is large and only small number is of interest. Also, it can be particularly useful to filter out categorical variable columns as there is no consistent ALE formulation and hence any results for categorical variables would be misleading."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/599/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>